recipes/cwl-utils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cwl-utils" %}
+{% set version = "0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cwl-utils-{{ version }}.tar.gz
+  sha256: 4ef2834aec167523c6079f78e42020aafbc88a697ca77449a3f85a6ddd243095
+
+build:
+  # Copied from schema-salad
+  skip: true  # [py2k or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - cwl-upgrader >=1.2
+    - packaging
+    - python
+    - requests
+    - schema-salad >=8.2,<9
+
+test:
+  imports:
+    - cwl_utils
+  commands:
+    - pip check
+  requires:
+    - pip
```

Comment:
You should run the tests as well

Reply:
You mean calling each of `cwl-cite-extract`, `cwl-docker-extract`, `cwl-expression-refactor`, `cwl-graph-split`, `cwl-normalizer` to verify they were correctly added to the `$PATH`?

Reply:
Added to the recipe the commands above.

Reply:
More about calling pytest post install https://github.com/conda-forge/cwltool-feedstock/blob/7640f7725c13c5b2c16be1c1037723dc5d9c761f/recipe/meta.yaml#L58

Reply:
Maybe add a `-k not ( tests_names requiring_cwlformat here )`

Reply:
Ah, I see. Aren't we running the tests already in the `cwl-utils` repository? On Cylc we ran the tests only on the CI with GH Actions, and in Conda just the build for Conda and tests for imports & scripts only.

Reply:
Running the tests here can catch conda packaging specific issues

Reply:
`python -m pytest -n auto --pyargs cwl_utils -k "not test_graph_split"` 

FYI, this doesn't editable installs (`pip install -e .`)